docs: update images using latest UI screenshots (#1992)
Unleash/unleash: website/docs/how-to/how-to-create-and-assign-custom-project-roles.md

1. Navigate to the custom project roles page by using the admin menu (the gear symbol) and navigating to users.
    ![A visual representation of the current step: the Unleash Admin UI with the steps highlighted.](/img/create-cpr-step-1.png)

> 2. Navigate to the "Project roles" tab.

Author: ```suggestion
2. Navigate to the "project roles" tab.
```

To assign a custom project role to a user:
1. Navigate to the project you want to assign the user a role in.
    ![The steps to navigate to a project: use the 'projects' navigation item and select your project.](/img/assign-cpr-step-1.png)
2. Navigate to the project's _access_ tab.
    ![A project overview with the 'access' tab highlighted.](/img/assign-cpr-step-2.png)
3. This step depends on whether the user has already been added to the project or not:

>     - If the user has already been added to the project, click on the edit icon coresponding with its line and from the overlay that will show up select the new role you want to assign it from the dropdown and save the changes.

Reviewer: ```suggestion
    - If the user has already been added to the project: Use the edit button in the user's row. Select the role you want to assign to the user, and then save your changes.
```

        ![A list of users with access to the current project. To the right of each user is a dropdown input labeled role.](/img/assign-cpr-step-3a.png)

>     - If the user _hasn't_ been added to the project, add them using the button 'Assing user/group'. From the overlay that will show up select the user, assign it a role and save the changes. Now you should be able to see the new user in the table.

Reviewer: ```suggestion
    - If the user _hasn't_ been added to the project, add them using the button labeled 'Assign user/group'. Use the form to select the user and the desired role, and then save your changes. The user will appear in the users table.
```
